build/Makefile.components

```diff
@@ -56,10 +56,15 @@ COREPROVIDERMODULE_STATICLIB_DEPS = \
 	omprovider
 
 # Foreach XYZ in the list above, reference $(SCXPAL_TARGET_DIR)/libXYZ.a
-SCXPAL_STATICLIB_DEPFILES = \
+SCXPAL_STATICLIB_DEPFILES_NODUPS = \
 	$(addprefix $(SCXPAL_TARGET_DIR)/lib, $(addsuffix .$(PF_STAT_LIB_FILE_SUFFIX), $(SCXPAL_STATICLIB_DEPS))) \
```

Comment:
Maybe put the result of this line in a variable since it it used three times. It would avoid future bugs where only one of them is modified.